pemFioi/quickAlgo/blockly_interface.js

```diff
@@ -507,6 +516,10 @@ function getBlocklyInterface(maxBlocks, nbTestCases) {
             if (this.mainContext.loadPrograms) {
                this.mainContext.loadPrograms(xml);
             }
+
+            if (this.quickAlgoInterface.options.canEditSubject) {
```

Comment:
It shouldn't be up to blockly_interface to check this option, but up to quickAlgoInterface.